amida-tech/amida-auth-microservice
# Feature/add UUID

If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What's this PR do?

Adds a new column for uuid, populates the said column with new uuid for existing users and will default and generate a new uuid on user creation. Existing response data will now include the new uuid. Migration/column additions will happen automatically.
 
#### Related JIRA tickets:

- https://jira.amida.com/browse/ORANGE-481

#### How should this be manually tested?

Run `yarn` to add new packages. Make sure your postgres user being user is a superuser. If the user is not, run this sql command for the user you are user: `ALTER USER user WITH SUPERUSER;`

#### Any background context you want to provide?

Support new microservices using uuid versus username (which is typically registration email)
